### ISSUE TITLE
How to DualBoot

### ISSUE THREAD

**Issue by morg-mov** [vicr123/theshell#51]
Is there any way to Dualboot Windows 10 and theShell OS?

**Comment by ItsJamie9494** [comment]
Yes. I would recommend installing Arch Linux alongside Windows 10, and then installing theShell on Arch Linux

**Comment by TheTechRobo** [comment]
Just open up theShell OS installer and click 'Open partition manager'. Click the Win 10 partition and press the 'resize partition' button. Resize it to your choice. Then click 'new partition', create new partition, and then click the new partition. Type mount point /

Works for me
